src/main/java/seedu/address/model/client/ClientEmail.java

```diff
@@ -50,7 +50,7 @@ public ClientEmail(String email) {
     public static class EmptyEmail extends ClientEmail {
         public static final ClientEmail EMPTY_EMAIL = new EmptyEmail();
         public EmptyEmail() {
-            super("");
+            super("cs2103@nus.edu");
         }
```

Comment:
Might be better to use `emptyemail@emptyemail.com` or something else that user is not likely to accidentally input (e.g. is a user accidentally enters cs2103@nus.edu instead of cs2103@u.nus.edu)?